package.json

```diff
@@ -4,7 +4,7 @@
   "description": "The semantic version parser used by npm.",
   "main": "semver.js",
   "scripts": {
-    "test": "standard && tap test/*.js --cov -J"
+    "test": "standard bin/semver *.js **/*.js && tap test/*.js --cov -J"
```

Comment:
Can you move the standard command invocation to a `posttest` script instead?

I prefer to run tests _before_ linting, so that it's less annoying when building new features or fixing bugs, and the `&&` is a sh-ism that doesn't work on Windows.

Reply:
Makes sense to me, working on it now.